src/NServiceBus.Metrics/ProbeBuilders/ReceivePerformanceDiagnosticsBehavior.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus.Pipeline;
+
+class ReceivePerformanceDiagnosticsBehavior : IBehavior<IIncomingPhysicalMessageContext, IIncomingPhysicalMessageContext>
```

Comment:
The core has also a behavior with the same name. Did we make sure that it does not collapse?